pyproject.toml

```
@@ -12,7 +12,7 @@ documentation = "https://docs.saleor.io/"
   [tool.poetry.dependencies]
   python = "~3.9"
   babel = ">=2.8,<2.13"
-  boto3 = "^1.28"
+  boto3 = "1.28.30"
```

Probably accidental change:

```suggestion
  boto3 = "^1.28"
```